pom.xml

```diff
@@ -68,6 +68,30 @@
                 <artifactId>gson</artifactId>
                 <version>2.10.1</version>
             </dependency>
+            <!--
+                Upgrade log4j version to fix security vulnerability
+                ref - https://logging.apache.org/log4j/2.x/security.html
+            -->
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-api</artifactId>
+                <version>2.17.1</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-core</artifactId>
+                <version>2.17.1</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-slf4j-impl</artifactId>
+                <version>2.17.1</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-1.2-api</artifactId>
+                <version>2.17.1</version>
+            </dependency>
         </dependencies>
```

Comment:
Please ensure all these were dependencies before too. If it is not, we don't have to bring it in.

Reply:
As discussed offline, these are declared in dependencyManagement and we just specify the version to be used for dependencies as well as transitive dependencies.